src/data/businessCase.ts

```diff
@@ -80,9 +73,27 @@ export const prepareBusinessCaseForApp = (
   };
 
   businessCase.lifecycleCostLines.forEach((line: any) => {
-    lifecycleCostLines[line.solution as keyof lifecycleCostLinesType][
-      yearMap[line.year]
-    ].push({ phase: line.phase, cost: line.cost });
+    const solution = (solutionName => {
+      switch (solutionName) {
+        case 'As Is':
+          return lifecycleCostLines['As Is'];
+        case 'Preferred':
+          return lifecycleCostLines.Preferred;
+        case 'A':
+          return lifecycleCostLines.A;
+        case 'B':
+          return lifecycleCostLines.B;
+        default:
```

Comment:
This fixes the "gotcha" that caused things not to work. If the switch doesn't match any of the known solution types, it will return `null` and skip it. This will protected us from the base case, which is `''` as well as any unknown types and/or malformed data.